source/controlTypes.py

```diff
@@ -157,6 +157,7 @@
 ROLE_ARTICLE = 150
 ROLE_REGION = 151
 ROLE_FIGURE = 152
+ROLE_MARKED_CONTENT = 153

```

Comment:
In the role constants, number 68 is missing. We could consider filling up that gap.
Also, I think that all roles consisting of multiple words, image map, input window, editable text don't have an underscore between the words. Could we keep that convention?

```suggestion
ROLE_MARKEDCONTENT = 153
```

Reply:
We already have ROLE_INSERTED_CONTENT and ROLE_DELETED_CONTENT. I'd prefer to keep that pattern, especially for these roles that are representing a particular property on content (such as inserted, deleted or marked) rather than say an actual control.

Reply:
Sorry @leonardder, I'm going to have to agree with Mick here. I think the extra underscore is more clear. I'd prefer not to have special rules about which words get underscores and which not.